### PR TITLE
Implement BSI VA

### PR DESCRIPTION
**Pull request by andykim**
## Description

Implement BSI VA

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Cosmetic change (text changing or color changing)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code (if necessary), particularly in hard-to-understand areas
- [ ] My changes generate no new vulnerabilities
- [ ] I have added tests that prove my fix is effective or that my feature works

## Screenshot
Screenshot from 2022-02-14 17-25-10

### CODE REVIEW

File: Xendit/M2Invoice/view/frontend/templates/multishipping/bsiva.phtml [hunk 1]
```diff
@@ -11,12 +11,12 @@
...
             var paymentMethodData = {
-                method: 'dp_mlhuillier'
+                method: 'bsiva'
             };
             layout([
                 {
-                    component: 'Xendit_M2Invoice/js/view/payment/method-renderer/multishipping/dp_mlhuillier',
-                    name: 'payment_method_dp_mlhuillier',
+                    component: 'Xendit_M2Invoice/js/view/payment/method-renderer/multishipping/bsiva',
+                    name: 'payment_method_bsiva',
```
Comment (IreneGohtami) [inline Xendit/M2Invoice/view/frontend/templates/multishipping/bsiva.phtml:19]: you're replacing `mlhuiller.phtml`?

**Review by IreneGohtami — COMMENTED** [review]
Minor comments, can you also update the readme.md?

File: Xendit/M2Invoice/view/frontend/templates/multishipping/bsiva.phtml (hunk 1 above)
Comment (andykim) [reply Xendit/M2Invoice/view/frontend/templates/multishipping/bsiva.phtml:19]: ah no. that's is the duplicated file. it has correct file name is dp_mlhuiller.phtml

**Review by IreneGohtami — APPROVED** [review]

**Review by kevinalfianto — APPROVED** [review]
LGTM

**Pull request merged**